assets/css/style.css

```diff
@@ -258,11 +258,13 @@ body {
 }
 
 .post-image {
```

Comment:
- [ ] Good work implementing  what the previews reviewer mentioned, however for the popup to match the Figma design as requested by the previous reviewer your images should display, kindly fix the  images and logos so they look like the Figma design 

![Screenshot_20220720_185255](https://user-images.githubusercontent.com/25227022/180050442-e78482e7-06f4-47d8-a131-f46e1cf6e823.png)